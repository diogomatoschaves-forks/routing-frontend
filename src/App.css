
body {
  font-size: 20px;
}

.custom-marker {
  color: #007a93;
  font-size: 3.6em !important;
  /* box-shadow: 0px 17px 5px 0px rgba(0,0,0,0.75); */
}

