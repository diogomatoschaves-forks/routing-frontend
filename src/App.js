import React, { Component } from "react"
import { Grommet, Box, Button } from "grommet"
import styled from "styled-components"

const theme = {
  global: {
    colors: {
      brand: "#228BE6"
    },
    font: {
      family: "Roboto",
      size: "20px",
      height: "20px"
    }
  }
}

const StyledButton = styled(Button)`
  border-radius: 20px;
`

const AppBar = props => (
  <Box
    tag="header"
    direction="row"
    align="center"
    justify="between"
    background="brand"
    pad={{ left: "medium", right: "small", vertical: "small" }}
    elevation="xlarge"
    style={{ zIndex: "1" }}
    {...props}
  />
)

class App extends Component {
  render() {
    const { option } = this.state

    return (
      <Grommet theme={theme}>
        <AppBar>Hello World!</AppBar>
        <StyledButton label="Edit" option={option} onClick={() => {}} />
      </Grommet>
    )
  }
}

export default App
