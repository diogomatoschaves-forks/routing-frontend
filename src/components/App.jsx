import React, { Component } from 'react'
import { Box, Button, Grommet, ResponsiveContext } from 'grommet'
import styled from 'styled-components'
import Map from './Map'
import Panel from './Panel'

const AppWrapper = styled.div`
  width: 100vw;
  height: 100vh;
`

class App extends Component {
  state = {
    // showSidebar: false
  }

  render() {
    return (
      <AppWrapper>
        <Panel />
        <Map />
      </AppWrapper>
    )
  }
}

export default App
