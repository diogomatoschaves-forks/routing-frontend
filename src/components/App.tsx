import React, { Component } from 'react'
import styled from 'styled-components'
import Map from './Map'
import Panel from './Panel'
import '../App.css'
// import 'antd/dist/antd.css';

const AppWrapper: any = styled.div`
  width: 100vw;
  height: 100vh;
`

class App extends Component {
  
  state = {}

  public render() {

    return (
      <AppWrapper>
        <Panel />
        <Map />
      </AppWrapper>
    )
  }
}

export default App
