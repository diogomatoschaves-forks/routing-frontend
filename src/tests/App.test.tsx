import React from 'react'
import TestRenderer from 'react-test-renderer'
import App from '../components/App'

// jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
//   GeolocateControl: jest.fn(),
//   Map: jest.fn(() => ({
//     addControl: jest.fn(),
//     on: jest.fn(),
//     remove: jest.fn()
//   })),
//   NavigationControl: jest.fn()
// }));

it('expect to render App component', () => {
  const testInstance = TestRenderer.create(<App/>)
  expect(testInstance.toJSON()).toMatchSnapshot()
}) 
