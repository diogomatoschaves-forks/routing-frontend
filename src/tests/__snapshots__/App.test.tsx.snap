// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render App component 1`] = `
<div
  className="sc-dnqmqq gqcTDv"
>
  0
  <div
    className="sc-gzVnrw kPKqNa"
  >
    <div
      className="sc-bwzfXH islmOK"
      direction="column"
    >
      <div
        className="sc-bwzfXH iSONxz"
        direction="row"
      >
        <div
          className="sc-htpNat eEBUlg"
          color="#e1e1e1"
        >
          <div
            className="sc-bwzfXH dmJOrF"
            height="100%"
          >
            <i
              aria-hidden="true"
              className="car icon sc-bxivhb vcXGv"
              fontSize="25px"
              height="20px"
              onClick={[Function]}
              overridecolor="#007a93"
              padding="0 0 0 0"
              position="absolute"
            />
          </div>
        </div>
        <div
          className="sc-ifAKCX dquRAm"
          height="38px"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          width="38px"
        >
          <div
            className="sc-bwzfXH hETasM"
            height="100%"
          >
            <i
              aria-hidden="true"
              className="male large icon sc-bxivhb hgrlEC"
              onClick={[Function]}
              overridecolor="#a6cad8"
              padding="0 0 0 0"
              position="absolute"
            />
          </div>
        </div>
      </div>
      <div
        className="ui divider sc-htoDjs lgjrUI"
      />
      <div
        className="sc-bwzfXH fOQsml"
        direction="row"
      >
        <div
          className="sc-htpNat iOFJBU"
          color="#007a93"
        >
          <div
            className="sc-bwzfXH gzsoVo"
            height="100%"
          >
            <i
              aria-hidden="true"
              className="map marker alternate icon sc-bxivhb gUqPoa"
              fontSize="25px"
              height="20px"
              onClick={[Function]}
              overridecolor="white"
              padding="0 0 0 0"
              position="absolute"
            />
          </div>
        </div>
        <div
          className="ui fluid input sc-bZQynM dlgHZH"
        >
          <input
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Origin"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="sc-bwzfXH fOQsml"
        direction="row"
      >
        <div
          className="sc-htpNat iOFJBU"
          color="#007a93"
        >
          <div
            className="sc-bwzfXH gzsoVo"
            height="100%"
          >
            <i
              aria-hidden="true"
              className="map marker icon sc-bxivhb gUqPoa"
              fontSize="25px"
              height="20px"
              onClick={[Function]}
              overridecolor="white"
              padding="0 0 0 0"
              position="absolute"
            />
          </div>
        </div>
        <div
          className="ui fluid input sc-bZQynM dlgHZH"
        >
          <input
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Destination"
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-bdVaJa jthvZh"
  />
</div>
`;
